scripts/build-all.sh

```diff
@@ -67,6 +83,6 @@ for prj in $PROJECTS; do
       ;;
   esac
   RETVAL=$?
-  [ $RETVAL -ne 0 ] && echo "Failure in project $prj" && exit -1
+  [ $RETVAL -ne 0 ] && echo "Failure in project $prj" && exit 1
```

Comment:
What does a -1 return code mean? I don't think it's valid. `exit 1` will indicate the script failed.

Reply:
Wow. That rabbit hole goes deep. Looks like this comes from `C`, where `resultCode < 0 ` was an error and `resultCode >= 0` had particular 'success' meanings.

Reply:
Well, that's not what bash uses though. In bash scripts, return code of 0 means success and 1-255 indicates error.